tuna/rocmlir/import_configs.py

```diff
@@ -28,141 +28,27 @@
 import os
 import logging
 import argparse
-import itertools
+import sys
 
 from sqlalchemy.exc import IntegrityError
 from tuna.dbBase.sql_alchemy import DbSession
 from tuna.utils.db_utility import connect_db
 from tuna.utils.logger import setup_logger
-from tuna.rocmlir.rocmlir_tables import RocMLIRDBTables
-
-## Adapted from perfRunner.getConvConfigurations.
```

Comment:
These are moved to rocmlir_tables.py, where they're also attached to conv- and gemm-specific classes.